### ISSUE TITLE
Template generation no longer working since updating to 3.0.0 and above

### ISSUE THREAD

**Issue by Witted** [mulesoft-labs/raml-dotnet-tools#239]
Hello,

Since updating to 3.0.1 and moving to VS2019 I am unable to generate boiler plate code using the scaffolder and the templates. 

When clicking "Edit Raml/OAS Properties" on the t4 template file I get the below error.
![image](https://user-images.githubusercontent.com/8886695/71665362-862fbe00-2d54-11ea-868b-61cd36866339.png)

Current RAML directory layout
![image](https://user-images.githubusercontent.com/8886695/71665557-5af99e80-2d55-11ea-9185-7f15df96f0a2.png)

Any help or indication as to why this is not working would be appreciated.

VS2019- version 16.4.2
Raml tool - 3.0.1

**Comment by woodp** [comment]
I tried a few different scenarios but could not reproduce this error. 

Please let me know all of the information of what you are doing. i.e. are you adding a RAML contract and then generating ? is the code generated or not ? what kind of project are you using ? is it a .net or .net core ?

Also, why are you trying to edit the RAML properties ? that is not a very common case, so, what are you trying to achieve ?

**Comment by Witted** [comment]
When trying to get this to work I tried a basic model i.e
![image](https://user-images.githubusercontent.com/8886695/71678638-8d1ef680-2d7d-11ea-83db-ab43ba0aca77.png)

The code is not being generated, the .raml file saves and does nothing more.

Framework is .Net 4.6.2

I am not trying to edit the properties, sometimes the raml does not auto-generate on save and just saving the properties (without changing them) has proven to be a more reliable way of generating the code for my project.

I included that error more because it crashes VS2019 completely more than how directly linked it is to the code not being generated.
